app-backend/db/src/main/resources/migrations/V5__Task_and_Task_Actions.sql

```diff
@@ -0,0 +1,33 @@
+-- Create tasks table
+
+CREATE TYPE task_status AS ENUM (
+    'UNLABELED', 'LABELING_IN_PROGRESS', 'LABELED', 'VALIDATION_IN_PROGRESS', 'VALIDATED'
+);
+
+CREATE TABLE tasks (
+  id uuid primary key,
+  created_at timestamp without time zone  not null,
+  created_by text not null references users (id),
+  modified_at timestamp without time zone not null,
+  modified_by text not null,
+  project_id uuid not null,
+  project_layer_id uuid not null,
+  status task_status not null,
+  locked_by text references users (id),
+  locked_on timestamp without time zone,
+  geometry geometry(Geometry, 3857),
```

Comment:
Wondering if there is a reason why we want to make this field nullable? If tasks are exclusively for annotate app, then I think tasks will always have geometries attached?

Reply:
I guess for the same reason that geometries are mysteriously nullable everywhere -- I'll change

Reply:
The better reason is that `GeoJSONFeature` requires geometry to be nullable, but it turned out I wasn't using that inheritance anyway, so I just removed it.